gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveDataset.java

```diff
@@ -119,244 +83,52 @@ public HiveDataset(FileSystem fs, HiveMetastoreClientPool clientPool, Table tabl
     this.table = table;
     this.properties = properties;
 
-    this.tableLocations = parseLocationIntoPaths(this.table.getSd().getLocation());
+    this.tableLocations = parseLocationIntoPaths(this.table.getDataLocation());
 
     this.tableRootPath = this.tableLocations.size() == 1 ?
         Optional.of(PathUtils.deepestNonGlobPath(this.tableLocations.get(0))) : Optional.<Path>absent();
 
-    this.relocateDataFiles =
-        Boolean.valueOf(properties.getProperty(RELOCATE_DATA_FILES_KEY, DEFAULT_RELOCATE_DATA_FILES));
-    this.targetTableRoot = properties.containsKey(COPY_TARGET_TABLE_ROOT) ?
-        Optional.of(resolvePath(properties.getProperty(COPY_TARGET_TABLE_ROOT))) : Optional.<Path>absent();
-
     this.tableIdentifier = this.table.getDbName() + "." + this.table.getTableName();
-  }
-
-  /**
-   * Takes a path with tokens {@link #databaseToken} or {@link #tableToken} and replaces these tokens with the actual
-   * database names and table name. For example, if db is myDatabase, table is myTable, then /data/$DB/$TABLE will be
-   * resolved to /data/myDatabase/myTable.
-   */
-  private Path resolvePath(String pattern) {
-    pattern = pattern.replaceAll(databaseToken, this.table.getDbName());
-    if (pattern.contains(tableToken)) {
-      pattern = pattern.replaceAll(tableToken, this.table.getTableName());
-      return new Path(pattern);
-    } else {
-      return new Path(pattern, this.table.getTableName());
-    }
+    log.info("Created Hive dataset for table " + tableIdentifier);
   }
 
   /**
    * Parse a location string from a {@link StorageDescriptor} into an actual list of globs.
    */
-  private List<Path> parseLocationIntoPaths(String sdLocation) {
+  protected static List<Path> parseLocationIntoPaths(Path sdLocation) {
```

Comment:
removed this method. It turns out the location is a `Path`, but `Path` is a very generic thing in Hadoop, and it can include globs, etc.
Instead of doing this, I am just checking that path is not a glob.